components/form/Checkbox.vue

```diff
@@ -127,6 +127,7 @@ export default {
         role="checkbox"
       />
       <span
+        v-if="label || labelKey || tooltipKey || tooltip"
```

Comment:
this should also check if the `label` slot is provided, think it's something like $slots.label